scipy/stats/stats.py

```diff
@@ -1805,6 +1810,18 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
     >>> extrapoints
     3
 
+    Generate histogram of cumulative frequencies
+    >>> cumfreqs, lowlim, binsize, extrapoints = stats.cumfreq(x)
+    >>> cumfreqs
+    array([ 3.,  3.,  3.,  4.,  4.,  4.,  5.,  5.,  5.,  6. ])
+
+    Plot cumulative frequencies
+    >>> a = np.arange(1, 11)
```

Comment:
This falls a bit out of the sky. Can you construct the horizontal axis for your plot from the ``lowlim`` and ``binsize`` return values?